soundcheck/src/app/globals.css

```
@@ -1,228 +1,59 @@
-}
-
-@keyframes pulse-soft {
-  0%,
-  100% {
-    transform: scale(1);
-    opacity: 1;
+@import "tailwindcss";
+@import "tw-animate-css";
+@import "@mcpjam/design-system/tokens.css";
+
+@source "../../../design-system/src/**/*.tsx";
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)